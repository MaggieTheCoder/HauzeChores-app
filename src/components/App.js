import React from "react";
import LoginPage from "./LoginPage";
import "../styles/App.css";

const App = () => {
  return (
    <div className="App">
      <header className="App-header">
        <h2>Hauze</h2>
      </header>
      <LoginPage />
    </div>
  );
};

export default App;
