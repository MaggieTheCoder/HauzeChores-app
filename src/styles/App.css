.App-header{
    display: flex;
    flex-direction: column;
    justify-content: center;
    align-items: center;
    border-bottom: 2px solid black;
}

.App-header h1{
    font-size: 70px;
    font-weight: 700;
    margin-bottom: 20px;
    margin-top: 10px;
    letter-spacing: -2px;
}
.App-header h3{
    margin-top: 20px;
    margin-bottom: 20px;
}

