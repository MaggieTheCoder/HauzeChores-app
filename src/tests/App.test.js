import React from "react";
import { render } from "@testing-library/react";
import App from "../components/App";

test("renders learn react link", () => {
  const { asFragment } = render(<App />);

  expect(asFragment()).toMatchSnapshot();
});
