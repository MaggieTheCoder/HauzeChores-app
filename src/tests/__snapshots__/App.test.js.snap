// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders learn react link 1`] = `
<DocumentFragment>
  <div
    class="App"
  >
    <nav>
      <div>
        <p>
          Logged in as 
        </p>
        <button
          type="button"
        >
          Log out
        </button>
      </div>
    </nav>
    <header
      class="App-header"
    >
      <h1>
        Houze
      </h1>
      <h3>
        Just get it done!
      </h3>
    </header>
    <div
      class="Login"
    >
      <form
        class="Login__login"
      >
        <input
          data-testid="login-email"
          name="email"
          placeholder="email"
          type="email"
        />
        <input
          data-testid="login-password"
          name="password"
          placeholder="password"
          type="password"
        />
        <button
          class="Login__button"
          type="submit"
        >
          Login
        </button>
      </form>
      <hr />
      <form
        class="Login__register"
      >
        <input
          data-testid="register-email"
          name="email"
          placeholder="email"
          type="email"
        />
        <input
          data-testid="register-password"
          name="password"
          placeholder="password"
          type="password"
        />
        <button
          class="Login__button"
          type="submit"
        >
          Register
        </button>
      </form>
    </div>
  </div>
</DocumentFragment>
`;
